### PR TITLE
Cds 72509 Revert PR step

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Added details about the Revert PR step

### CODE REVIEW

**Review by manavjot-harness — DISMISSED** [review]

File: docs/continuous-delivery/gitops/harness-git-ops-application-set-tutorial.md [hunk 1]
```diff
@@ -444,6 +444,19 @@ In **Execution**, Harness automatically adds the following steps:
   ![](./static/harness-git-ops-application-set-tutorial-56.png)
 
 * **Merge PR**: Merges the new PR.
+* **Revert PR**: Reverts the commit passed and creates a new PR. Use this step if you want to run any tests or automation on the pipeline, and revert the commit done by the Update Release Repo step.
+
```
Comment (michaelcretzman) [inline docs/continuous-delivery/gitops/harness-git-ops-application-set-tutorial.md:448]:
```suggestion
* **Revert PR**: Reverts the commit passed and creates a new PR. Use this step if you want to run any tests or automation on the pipeline and then revert the commit done by the Update Release Repo step.
  
```

File: docs/continuous-delivery/gitops/harness-git-ops-application-set-tutorial.md [hunk 2]
```diff
@@ -444,6 +444,19 @@ In **Execution**, Harness automatically adds the following steps:
...
 * **Merge PR**: Merges the new PR.
+* **Revert PR**: Reverts the commit passed and creates a new PR. Use this step if you want to run any tests or automation on the pipeline, and revert the commit done by the Update Release Repo step.
+  
+  The Revert PR step uses the commitId of the Update Release Repo step as input. The commitId can be an expression, runtime input, or a static value. For example, `<+pipeline.stages.deploy.spec.execution.steps.updateReleaseRepo.updateReleaseRepoOutcome.commitId>`. 
+  
+  The Revert PR step creates a new branch and creates a commit to revert the changes done in the Update Release Repo step commit. 
+  
+  You can create another Merge PR step to merge the Revert PR step. 
+
+  :::info limitation
```
Comment (michaelcretzman) [inline docs/continuous-delivery/gitops/harness-git-ops-application-set-tutorial.md:455]:
```suggestion
  :::info Limitation
```

**Review by michaelcretzman — DISMISSED** [review]
Approved with a couple suggestions.

**Review by shivagowda — APPROVED** [review]
lgtm

**Comment by shivagowda** [comment]
@michaelcretzman this looks good, please merge

**Pull request merged**